.changeset/eighty-planets-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-expo': patch
+---
+
+Update `@clerk/clerk-expo` dependencies.
```

Comment:
So with changesets it'll already be clear that the `@clerk/clerk-expo` package will be updated. So that information is a duplicate then really.

I'd write it like this:

```suggestion
Update `expo-auth-session` dependency from 5.0.2 to 5.4.0 and `expo-web-browser` from 12.3.2 to 12.8.2
```

Reply:
Thank's for the suggestion!